libmscore/layout.cpp

```diff
@@ -3784,7 +3784,7 @@ System* Score::collectSystem(LayoutContext& lc)
                   Measure* m = toMeasure(lc.curMeasure);
                   if (firstMeasure) {
                         layoutSystemMinWidth = minWidth;
-                        system->layoutSystem(minWidth);
+                        system->layoutSystem(minWidth, lc.firstSystem, lc.firstSystemIndent);
```

Comment:
These two added parameter should be added indeed. Only in <code>layoutlinear</code> they don't make sense.